Feature/upload file kunisu

kuniokuniyama opened the pull request:
### 投稿一覧


### TODO

- [x] ファイル投稿、取得


### 関連PR
- なし

ifmmatsumoto commented on spring_prj/src/main/java/com/c4ccup/laugh/controller/MyPageController.java:
```diff
@@ -121,26 +122,7 @@ private int[] getActiveTerm(LocalDate debutAt) {
+    private ResponseEntity<ApiResource<Messages>> uploadContent(@Validated @RequestBody MyPageBean bean) {
 
     //コンテンツをS3に登録
     if(bean.getContent() != null && !bean.getContent().isEmpty()) {
```
バリデーションをしているのでチェックは不要

ifmmatsumoto replied:
インデントをきれいにする

ifmmatsumoto replied:
https://c4claugh.s3.ap-northeast-1.amazonaws.com/はawsS3Utilで定数化しているのでそれを使う

ifmmatsumoto replied:
解説してほしい
```
    StringBuilder title = new StringBuilder();
    for (int i = 0; i < bean.getTitle().split("\\.").length - 1; i++) {
        title.append(bean.getTitle().split("\\.")[i]);
        if (i < bean.getTitle().split("\\.").length - 2) {
            title.append(".");
        }
    }
```

kuniokuniyama committed:
Merge branch 'develop' of https://github.com/toritakuro/laugh_api into feature/upload_file_kunisu

kuniokuniyama committed:
レビュー内容の修正
・@Validatedがあるので、if文でのnullチェックやemptyチェック箇所を削除
・インデントを整理(反省します)
・AWSのURLは定数化されている箇所は定数を使用
・タイトルの拡張子以前取得方法は、ループではなくlastIndexOfを使用

kuniokuniyama committed:
Merge branch 'develop' of https://github.com/toritakuro/laugh_api into feature/upload_file_kunisu

kuniokuniyama committed:
投稿ファイル取得の際、コンストラクタを使ってレスポンスにデータをセットするよう修正

kuniokuniyama committed:
Merge branch 'develop' of https://github.com/toritakuro/laugh_api into feature/upload_file_kunisu
